src/board/board.ts

```diff
@@ -89,7 +93,23 @@ export const highlightAllowedTiles = (
     });
     return newBoard;
   } else {
-    return board;
+    const lastPlayedTile = getLastPlayedTile(board);
```

Comment:
`else` is not necessary here, because `if` return already something

Reply:
Done